src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/ExtensionMethodImportCompletionHelper.cs

```diff
@@ -119,39 +126,56 @@ internal static partial class ExtensionMethodImportCompletionHelper
 
         }
 
-        private static ImmutableArray<SerializableImportCompletionItem> ConvertSymbolsToCompletionItems(ImmutableArray<IMethodSymbol> extentsionMethodSymbols, CancellationToken cancellationToken)
+        private static ImmutableArray<SerializableImportCompletionItem> ConvertSymbolsToCompletionItems(
+            Compilation compilation, ImmutableArray<IMethodSymbol> extentsionMethodSymbols, ImmutableArray<ITypeSymbol> targetTypeSymbols, CancellationToken cancellationToken)
         {
+            Dictionary<ITypeSymbol, bool> typeConvertibilityCache = new();
```

Comment:
Does this need the SymbolEqualityComparer.Default?

Reply:
Reference equality is fine here, since all the symbols are retrieved from the same compilation/semantic model.